pysatSpaceWeather/instruments/methods/ace.py

```diff
@@ -154,10 +154,7 @@ def list_files(name, tag='', inst_id='', data_path=None, format_str=None):
         format_str = '_'.join(["ace", name, tag,
                                '{year:04d}-{month:02d}-{day:02d}.txt'])
 
-    if data_path is None:
-        raise ValueError('No path to data supplied through `data_path`')
```

Comment:
Need to update the docstring, as this error was removed.

Reply:
Updated docstring. As an FYI `from_os` still raises a ValueError if it gets a None but that raises should be applied there.